cadquery/cq.py

```diff
@@ -2858,7 +2842,7 @@ def clean(self):
         return self.newObject(cleanObjects)
 
     def text(self, txt, fontsize, distance, cut=True, combine=False, clean=True,
-             font="Arial", kind='regular'):
+             font="Arial", kind='regular',align='center',valign='center'):
```

Comment:
I don't know that it matters that much, but you might consider changing `align` to `halign` so that it mirrors the `valign` parameter.

Reply:
Agreed